akka-actor/src/main/scala/akka/io/TcpConnection.scala

```diff
@@ -189,7 +189,7 @@ private[io] abstract class TcpConnection(val tcp: TcpExt, val channel: SocketCha
 
   /** used in subclasses to start the common machinery above once a channel is connected */
   def completeConnect(registration: ChannelRegistration, commander: ActorRef,
-                      options: immutable.Traversable[SocketOption]): Unit = {
+    options: immutable.Traversable[SocketOption]): Unit = {
```

Comment:
no idea actually, I thought it would. I've given up on this PR for the time being though :/